webservices/legal_docs/load_legal_docs.py

```diff
@@ -377,7 +377,7 @@ def get_mur_names(mur_names={}):
                 mur_names[row[1]] = row[2]
     return mur_names
 
-def process_mur(mur):
+def process_murs(raw_mur_tr_element_list):
```

Comment:
This commit does not work. When I run the code, I get:
```
INFO:manager:Loading archived MURs
Traceback (most recent call last):
  File "manage.py", line 381, in <module>
    manager.run()
  File "/Users/radhakrishnanvrajmohan/git/openFEC/.env/lib/python3.6/site-packages/flask_script/__init__.py", line 412, in run
    result = self.handle(sys.argv[0], sys.argv[1:])
  File "/Users/radhakrishnanvrajmohan/git/openFEC/.env/lib/python3.6/site-packages/flask_script/__init__.py", line 383, in handle
    res = handle(*args, **config)
  File "/Users/radhakrishnanvrajmohan/git/openFEC/.env/lib/python3.6/site-packages/flask_script/commands.py", line 216, in __call__
    return self.run(*args, **kwargs)
  File "/Users/radhakrishnanvrajmohan/git/openFEC/webservices/legal_docs/load_legal_docs.py", line 446, in load_archived_murs
    process_murs(raw_mur_tr_element_list)
  File "/Users/radhakrishnanvrajmohan/git/openFEC/webservices/legal_docs/load_legal_docs.py", line 385, in process_murs
    = re.findall("<td[^>]*>(.*?)</td>", mur[2], re.S)
NameError: name 'mur' is not defined
```

Reply:
@vrajmohan I am not sure why and how this works on mine
./manage.py load_archived_murs -s 1252
INFO:manager:Loading archived MURs
INFO:manager:Loading archived MUR 1252
INFO:manager:Loading file 1252_A.pdf
INFO:manager:Loading file 1252_B.pdf
INFO:manager:Loading file 1252_C.pdf
INFO:manager:Loading file 1252_D.pdf
INFO:manager:Loading file 1252_E.pdf
INFO:manager:1 archived MURs loaded

Reply:
Try running it without specifying a specific archived MUR to load.